less/components/amount-buttons.less

```diff
@@ -64,6 +64,17 @@
   .amount-button {
     letter-spacing: -1;
     margin-bottom: 0;
+
+    .per-month-note {
+      display: inline-block;
+      font-size: 11px;
+      margin-left: 5px;
+      width: 33px;
```

Comment:
Could this actually be dynamic when there’s room? Many locales will have visual issues, this would help to at least fix the situation on large screens and/or when the amount + currency is short.

Reply:
Thinking about this more I wonder if we should always put the `per month` note on a separate line (instead of positioning it next to the `[money amount]`). We can't predict how long each localized string and converted currency string are going to be. There might be cases where among the 4 amount buttons we have, few have `[money amount]` and `per month` next to each other and the rest have the 2 strings on separate lines. Design wise this will look a bit odd in my opinion...

Reply:
tagging @xmatthewx for design feedback - I'm not sure if we wanna get the current design live first and worry about refinement after or ?

Reply:
@mmmavis – you're probably right. Putting per month on a second line will make it more flexible for localization. 

I used `display:block; margin: -4px 0 6px;` to tighten in up, but you see what you think is best. 

I wonder if we should handle the Other option in the same way so we can preserve the height (removing monthly from the placeholder and putting in the label).  
 

<img width="469" alt="image" src="https://user-images.githubusercontent.com/53411/46695453-943c7080-cbdd-11e8-875e-28518569fd2b.png">

Reply:
@alanmoo - feel free to chime in. I'm less familiar with this tool and the original request than you are...

Reply:
This makes sense to me, still fits the original need.

Reply:
Thanks! I'll update my PR and flag @TheoChevalier for review again.